shippable.yml

```diff
@@ -0,0 +1,81 @@
+
+language: c
+
+compiler:
+  - gcc
+  - clang
+
+env:
+  matrix:
+    - CONF=""
+    # - CONF="--disable-abi-compat"
+    # - CONF="--enable-schedule-sp"
+    # - CONF="--enable-schedule-iquery"
+    # - CONF="--enable-dpdk-zero-copy"
+    - CROSS_ARCH="arm64"
+    # - CROSS_ARCH="armhf" CFLAGS="-march=armv7-a"
+    # - CROSS_ARCH="powerpc"
+    # - CROSS_ARCH="i386"
+  global:
```

Comment:
Coverity I also thing should go away. This script does not send results to Coverity.